src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
```suggestion
    void draw();
```
Use System.out.println inside. 
_To 'draw' means to print out all information about a figure using System.out.println()_

Reply:
not fixed

Reply:
not fixed